src/js/post-911-gib-status/containers/Post911GIBStatusApp.jsx

```diff
@@ -31,15 +29,10 @@ function AppContent({ children, isDataAvailable }) {
 }
 
 class Post911GIBStatusApp extends React.Component {
-  componentDidMount() {
-    this.props.getEnrollmentData();
-  }
 
   render() {
     // TODO: change the service name below from "user-profile" to
```

Comment:
We should probably change this now that we have a service available for this app. Tried doing it quickly in this PR, but I think it's going to be a more extensive change, so saving that for a separate PR.